src/kirby/components/icon/kirby-icon-settings.ts

```diff
@@ -46,5 +41,17 @@ export const kirbyIconSettings: IconSettings = {
     { name: 'verify', svg: 'assets/kirby/icons/svg/verify.svg', unicode: 0xe927 },
     { name: 'warning', svg: 'assets/kirby/icons/svg/warning.svg', unicode: 0xe928 },
     { name: 'write-message', svg: 'assets/kirby/icons/svg/write-message.svg', unicode: 0xe929 },
+    { name: 'backspace', svg: 'assets/kirby/icons/svg/backspace.svg', unicode: 0xee954 },
```

Comment:
I have one really annoting request. Could you sort the objects alphabetic after the `name`?
It was before 😛

Reply:
Well, this is not used anymore, as native is going out. Is it worth it ;)

Reply:
nope not worth it. 😄